PRAGMA table_info returns Data type error on node.js environment

Hello,
I tried executing PRAGMA table_info using the node.js environment and it returned a Data type error. Other commands seem to work fine.

Code used (file main.js):
```javascript
var duckdb = require('duckdb');

var db = new duckdb.Database('database.duckdb');
var con = db.connect();

var n_rows = 20;

// Create a new table
console.log('Creating table.')
con.run('CREATE TABLE a (i INTEGER, j INTEGER)');
var stmt = con.prepare('INSERT INTO a VALUES (?, ?)');
for (var i = 0; i < n_rows; i++) {
  stmt.run(i, i*10);
}
stmt.finalize();

console.log('Table created, now getting its info.')
// Iterate over the tables and get their info
con.all('PRAGMA show_tables;', function(err, res){
  if (err){
    throw err;
  }
  res.forEach(e => {
    console.log(e['name'])
    con.all(`PRAGMA table_info(${e['name']})`, function(err, res){
      if (err){
        throw err;
      }
      console.log(res);
    });
  });
})
```

Command executed:
```bash
node main.js
```

Output:
```
Creating table.
Table created, now getting its info.
a
undefined:0
Error: Data type is not supported BOOLEAN
```
PRAGMA table_info seems to work fine when the table **is not** present. But, it returns an error with a table that **is** present.
I searched the issues page but couldn't find anything related. Is this a bug (or known issue), or is it a problem somewhere on my side?

From file you mentioned, I tried the following command:
```bash
./scripts/node_build.sh 15
```
(I am not sure about that 15).

Initially it would fail compilation because some unique_ptrs didn't have std:: in front of them. So I just added it.
Now, it fails during compilation because it cannot find the module utf8proc_wrapper.hpp
```
../src/duckdb.cpp:62605:10: fatal error: 'utf8proc_wrapper.hpp' file not found
```
I am not sure where I went wrong.

--- a/tools/nodejs/package.json
+++ b/tools/nodejs/package.json
@@ -1,7 +1,7 @@
 {
   "name": "duckdb",
   "main": "./lib/duckdb.js",
-  "version": "0.2.4",
+  "version": "0.3.2-dev593.0",
   "description": "DuckDB node.js API",
   "gypfile": true,
   "dependencies": {
@@ -25,8 +25,8 @@
     "test": "test"
   },
   "devDependencies": {
-    "mocha": "^8.3.0",
-    "aws-sdk": "^2.790.0"
+    "aws-sdk": "^2.790.0",
+    "mocha": "^8.3.0"
   },
   "repository": {
     "type": "git",

--- a/tools/nodejs/package.json
+++ b/tools/nodejs/package.json
@@ -1,7 +1,7 @@
 {
   "name": "duckdb",
   "main": "./lib/duckdb.js",
-  "version": "0.3.2-dev593.0",
+  "version": "0.2.4",
   "description": "DuckDB node.js API",
   "gypfile": true,
   "dependencies": {

--- a/tools/nodejs/src/statement.cpp
+++ b/tools/nodejs/src/statement.cpp
@@ -137,6 +137,7 @@ static Napi::Value convert_chunk(Napi::Env &env, std::vector<std::string> names,
 			Napi::Value value;
 
 			auto dval = chunk.GetValue(col_idx, row_idx);
+
 			if (dval.is_null) {
 				row_result.Set(node_names[col_idx], env.Null());
 				continue;
@@ -170,6 +171,9 @@ static Napi::Value convert_chunk(Napi::Env &env, std::vector<std::string> names,
 #endif
 			case duckdb::LogicalTypeId::VARCHAR: {
 				value = Napi::String::New(env, dval.str_value);
+			} break;
+			case duckdb::LogicalTypeId::BOOLEAN: {
+				value = Napi::Boolean::New(env, dval.value_.boolean);
 			} break;
 			case duckdb::LogicalTypeId::BLOB: {
 				value = Napi::Buffer<char>::Copy(env, dval.str_value.c_str(), dval.str_value.length());

--- a/tools/nodejs/src/statement.cpp
+++ b/tools/nodejs/src/statement.cpp
@@ -137,7 +137,6 @@ static Napi::Value convert_chunk(Napi::Env &env, std::vector<std::string> names,
 			Napi::Value value;
 
 			auto dval = chunk.GetValue(col_idx, row_idx);
-
 			if (dval.is_null) {
 				row_result.Set(node_names[col_idx], env.Null());
 				continue;

--- a/tools/nodejs/test/data_type_support.test.js
+++ b/tools/nodejs/test/data_type_support.test.js
@@ -1,0 +1,23 @@
+var sqlite3 = require("..");
+var assert = require("assert");
+
+describe("data type support", function () {
+  let db;
+  before(function (done) {
+    db = new sqlite3.Database(":memory:", done);
+  });
+
+  it("supports BOOLEAN values", function (done) {
+    db.run("CREATE TABLE boolean_table (i BOOLEAN)");
+    const stmt = db.prepare("INSERT INTO boolean_table VALUES (?)");
+    const values = [true, false];
+    values.forEach((bool) => {
+      stmt.run(bool);
+    });
+    db.prepare("SELECT i from boolean_table;").all((err, res) => {
+      assert(err === null);
+      assert(res.every((v, i) => v.i === values[i]));
+      done();
+    });
+  });
+});

